### ISSUE TITLE
Invalid position given by 'no-multiple-empty-lines'

### ISSUE THREAD

**Issue by gavmck** [AtomLinter/linter-eslint#1061]
ESLint returned a point that did not exist in the document being edited.
Rule: `no-multiple-empty-lines`
Requested range: 20:1

Debug information:
```json
{
  "atomVersion": "1.22.1",
  "linterEslintVersion": "8.4.0",
  "linterEslintConfig": {
    "lintHtmlFiles": false,
    "useGlobalEslint": false,
    "showRuleIdInMessage": true,
    "disableWhenNoEslintConfig": true,
    "eslintrcPath": "",
    "globalNodePath": "",
    "advancedLocalNodeModules": "",
    "eslintRulesDirs": [],
    "disableEslintIgnore": false,
    "disableFSCache": false,
    "fixOnSave": false,
    "scopes": [
      "source.js",
      "source.jsx",
      "source.js.jsx",
      "source.babel",
      "source.js-semantic"
    ],
    "rulesToSilenceWhileTyping": [],
    "rulesToDisableWhileFixing": [],
    "ignoreFixableRulesWhileTyping": false
  },
  "eslintVersion": "2.13.1",
  "hoursSinceRestart": 38.4,
  "platform": "darwin",
  "eslintType": "local project",
  "eslintPath": "/Users/gavynmckenzie/dev/php/cima-study/node_modules/eslint",
  "editorScopes": [
    "source.js",
    "punctuation.terminator.statement.js"
  ]
}
```

**Comment by Arcanemagus** [comment]
Please update the version of ESLint you are using ([v1 -> v2](https://eslint.org/docs/user-guide/migrating-to-2.0.0), [v2 -> v3](http://eslint.org/docs/user-guide/migrating-to-3.0.0), [v3 -> v4](http://eslint.org/docs/user-guide/migrating-to-4.0.0)), this particular rule was fixed in ESLint version [v3.5.0](http://eslint.org/blog/2016/09/eslint-v3.5.0-released).

Thanks for taking the time to report this though!

**Comment by gavmck** [comment]
Ace, thanks!
